### PR TITLE
fix: migrate to assets-pkg

### PR DESCRIPTION
**Pull request by masone**
https://autoricardo.atlassian.net/browse/CAR-8227

generating assets based on our conventions

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -76,15 +74,12 @@
     "postcss-easy-import": "^3.0.0",
     "postcss-loader": "^4.2.0",
     "react": "^16.14.0",
-    "react-docgen-typescript-loader": "^3.6.0",
```
Comment (masone) [inline package.json:79]: part of the latest storybook version

File: package.json [hunk 2]
```diff
@@ -76,15 +74,12 @@
...
-    "react-dom": "^16.12.0",
-    "react-test-renderer": "^16.12.0",
```
Comment (masone) [inline package.json:81]: was not really needed, built into testing-lib

File: package.json [hunk 3]
```diff
@@ -76,15 +75,12 @@
     "postcss-easy-import": "^3.0.0",
     "postcss-loader": "^4.2.0",
     "react": "^16.14.0",
```
Comment (GutierrezGo) [inline package.json:77]: just one open question. Is it possible to upgrade the react version as in the other projects?

**Review by GutierrezGo — COMMENTED** [review]
Sorry, one question do we need to move the icons into the assets/src as well?

**Comment by masone** [comment]
Thanks, I missed those! Will move them as well and ping you again for review

**Review by GutierrezGo — APPROVED** [review]
It looks good to me.
Thanks!! 🌈

**Review by talamcol — APPROVED** [review]
Looks good!

**Pull request merged**